src/plays/2048/Game2048Components/Game.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import Grid from "./Grid";
+import TranformGrid from "../transformGrid";
+
+function Game(props) {
+  const [grid, setGrid] = useState(() => {
+    let _grid = [
+      [null, null, null, null],
+      [null, null, null, null],
+      [null, null, null, null],
+      [null, null, null, null]
+    ];
+
+    _grid = TranformGrid.addRandomCell(_grid);
+    _grid = TranformGrid.addRandomCell(_grid);
+
+    return _grid;
+  });
+
+  return (
+    <div className="game">
+      <Grid
+        grid={grid}
+        setGridState={setGrid}
```

Comment:
@Shivam-Katare, Do not pass the setter function as a prop, instead create a handler and pass that function as a prop.

Example : 
```js
const handleGrid=(value)=>setGrid(value)
```